### PR TITLE
CCTRI 1348/write respond observables and trigger logic

### PR DESCRIPTION
**Pull request by GeorgiyII**

### CODE REVIEW

File: api/respond.py [hunk 1]
```diff
@@ -12,15 +14,62 @@
...
+        }
+    }
+
+
 @respond_api.route('/respond/observables', methods=['POST'])
 def respond_observables():
-    _ = get_jwt()
-    _ = get_observables()
-    return jsonify_data([])
+    relay_input = get_json(ObservableSchema(many=True))
```
Comment (gdavoian) [inline api/respond.py:45]: Why don't just to use `relay_input = get_observables()`? If you don't like the `get_observables` function from line 13 then just remove it from the code.

File: api/respond.py [hunk 2]
```diff
@@ -12,15 +14,62 @@
...
+        g.actions.append(extract_action(observable))
+
+    return jsonify_data(g.actions)
 
 
 @respond_api.route('/respond/trigger', methods=['POST'])
 def respond_trigger():
-    _ = get_jwt()
-    _ = get_action_form_params()
+    relay_input = get_json(ActionFormParamsSchema())
```
Comment (gdavoian) [inline api/respond.py:62]: The same here: `relay_input = get_action_form_params()`

Comment (GeorgiyII) [reply api/respond.py:58]: done

File: api/respond.py (hunk 1 above)
Comment (GeorgiyII) [reply api/respond.py:41]: done

**Pull request merged**